test/quantization/fx/test_quantize_fx.py

```diff
@@ -6555,6 +6555,37 @@ def forward(self, x):
         ])
         m3(*example_inputs)
 
+    def test_getitem_no_dequant_quant(self):
+        """
+        Test that we do not add dequant and quant ops around getitem.
+        """
+        class M(torch.nn.Module):
+            def __init__(self):
+                super().__init__()
+                self.conv1 = torch.nn.Conv1d(in_channels=5, out_channels=5, kernel_size=5, padding=0)
+                self.conv2 = torch.nn.Conv1d(in_channels=5, out_channels=5, kernel_size=5, padding=0)
+
+            def forward(self, inputs):
+                # inputs: [1, 5, 10]
+                x1 = self.conv1(inputs)
+                # x: [1, 5, 6]
+                x1 = x1 + inputs[:, :, -6:]
+                x2 = self.conv2(x1)
+                x2 = x2 + x1[:, :, -2:]
+                return x2
+
+        m = M()
+        m.eval()
+        qconfig_mapping = get_default_qconfig_mapping()
+        m = prepare_fx(m, qconfig_mapping, example_inputs=torch.rand(1, 5, 10))
```

Comment:
I think we may want to check the input and output observer inserted for getitem are the same